src/main/java/io/github/jhipster/config/JHipsterConstants.java

```diff
@@ -22,23 +22,20 @@
 /**
  * JHipster constants.
  */
-public final class JHipsterConstants {
+public interface JHipsterConstants {
```

Comment:
I thought this was only adding tests but seems you rewrote some stuff as well so I would like @jdubois to have a look as well before merging this

Reply:
Just some little tweaks, like those message constants and things like this which changes a class that’s never instantiated to an interface (helps with coverage).

Other than that just minor refactoring so I could spy or override selectively.

No functional changes I think.